packages/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -0,0 +1,228 @@
+        _parseRemoteConfigParameters(parameters: properties[_parametersKey]);
+    return remoteConfig;
+  }
+
+  static Map<String, RemoteConfigValue> _parseRemoteConfigParameters(
+      {Map<dynamic, dynamic> parameters}) {
+    final Map<String, RemoteConfigValue> parsedParameters =
+        <String, RemoteConfigValue>{};
+    parameters.forEach((dynamic key, dynamic value) {
+      final ValueSource valueSource = _parseValueSource(value['source']);
+      final RemoteConfigValue remoteConfigValue =
+          new RemoteConfigValue._(value['value'].cast<int>(), valueSource);
+      parsedParameters[key] = remoteConfigValue;
+    });
+    return parsedParameters;
+  }
+
+  static ValueSource _parseValueSource(String sourceStr) {
+    switch (sourceStr) {
+      case 'valueStatic':
+        return ValueSource.valueStatic;
+      case 'valueDefault':
+        return ValueSource.valueDefault;
+      case 'valueRemote':
+        return ValueSource.valueRemote;
+      default:
+        return ValueSource.valueStatic;
+    }
+  }
+
+  static LastFetchStatus _parseLastFetchStatus(String statusStr) {
+    switch (statusStr) {
+      case 'success':
+        return LastFetchStatus.success;
+      case 'failure':
+        return LastFetchStatus.failure;
+      case 'throttled':
+        return LastFetchStatus.throttled;
+      case 'noFetchYet':
+        return LastFetchStatus.noFetchYet;
+      default:
+        return LastFetchStatus.failure;
+    }
+  }
+
+  /// Set the configuration settings for this RemoteConfig instance.
+  ///
+  /// This can be used for enabling developer mode.
+  Future<void> setConfigSettings(
+      RemoteConfigSettings remoteConfigSettings) async {
+    await _channel
+        .invokeMethod('RemoteConfig#setConfigSettings', <String, dynamic>{
+      'debugMode': remoteConfigSettings.debugMode,
+    });
+    _remoteConfigSettings = remoteConfigSettings;
+  }
+
+  /// Fetches parameter values for your app. Parameter values may be from
+  /// Default Config (local cache) or Remote Config if enough time has elapsed
+  /// since parameter values were last fetched from the server. The default
+  /// expiration time is 12 hours. Expiration must be defined in seconds.
+  Future<void> fetch({Duration expiration: const Duration(hours: 12)}) async {
+    try {
+      final Map<dynamic, dynamic> properties = await _channel.invokeMethod(
+          'RemoteConfig#fetch',
+          <dynamic, dynamic>{'expiration': expiration.inSeconds});
+      _lastFetchTime = new DateTime.fromMillisecondsSinceEpoch(
+          properties[_lastFetchTimeKey]);
+      _lastFetchStatus = _parseLastFetchStatus(properties[_lastFetchStatusKey]);
+    } on PlatformException catch (e) {
+      _lastFetchTime =
+          new DateTime.fromMillisecondsSinceEpoch(e.details[_lastFetchTimeKey]);
+      _lastFetchStatus = _parseLastFetchStatus(e.details[_lastFetchStatusKey]);
+      if (e.code == RemoteConfig._fetchFailedThrottledKey) {
```

Comment:
I think you can leave off the `RemoteConfig.` here

Reply:
Done